psychopy/visual/window.py

```diff
@@ -383,6 +383,13 @@ def __del__(self):
         if self._closed!=False:
             self.close()
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        if self._closed != False:
```

Comment:
Is that logic the correct way around (confusing double negatives). Close the window if it closed==False. If closed!=False then closed==True and in that case we shouldn't close again, right?